integrations/object_store/Cargo.toml

```diff
@@ -19,20 +19,20 @@
 description = "OpenDAL's object_store Integration"
 name = "object_store_opendal"
 
-authors.workspace = true
-edition.workspace = true
-homepage.workspace = true
-license.workspace = true
-repository.workspace = true
-rust-version.workspace = true
-version.workspace = true
+authors = ["Apache OpenDAL <dev@opendal.apache.org>"]
+edition = "2021"
+homepage = "https://opendal.apache.org/"
+license = "Apache-2.0"
+repository = "https://github.com/apache/opendal"
+rust-version = "1.67"
+version = "0.42.0"
```

Comment:
0.45.0?

Reply:
We haven't release `object_store_opendal` for a while. It's current version is `0.42.0`.

Reply:
Got it.